src/prebid.js

```diff
@@ -194,7 +194,7 @@ $$PREBID_GLOBAL$$.setTargetingForAst = function() {
   targeting.setTargetingForAst();
 
   // emit event
-  events.emit(SET_TARGETING);
+  events.emit(SET_TARGETING, targeting.getAllTargeting());
```

Comment:
do we doc the event signatures anywhere? Probably needs to be done since we are adding params.

Reply:
Nowhere that I know of.  The events themselves are documented on the publisher API reference page, but it doesn't include details of what arguments the events pass to the event handlers.

I did add a new event `bidderDone` event though that should be added to that page.  Just created a pull-request: https://github.com/prebid/prebid.github.io/pull/695